.github/workflows/deployment.yaml

```diff
@@ -19,72 +25,93 @@ on:
           - "node"
           - "boot-api"
           - "sentinel"
-      network:
-        description: "Network to deploy to"
+      image:
+        description: "Image Version"
         required: true
         type: choice
+        default: "new"
         options:
-          - "Baobab"
-          - "Cypress"
-
+          - new
+          - por:v0.0.1.20240621.0703.82429c9:Update price pair tags
+          - vrf:v0.5.1.20240621.0710.dfa5c7d:Update price pair tags
```

Comment:
are these tags for the testing purpose?

Reply:
Yes, it was created for testing. 
we can remove it for fresh new one~